src/clients/saveMessage.js

```diff
@@ -1,18 +1,26 @@
-import Message from "../models/message.js";
+import { mainMessage, reserveMessage } from "../models/message.js";
 import lockedSync from "locked-sync";
+
 const sync = lockedSync();
+let mainDatabase = mainMessage
+let secondaryDatabase = reserveMessage
+
 
 export default async (messageParams) => {
-  const message = new Message(messageParams);
   const end = await sync();
+
+
   try {
 
-    const doc = await message.save();
+    const doc = await mainDatabase.create(messageParams)
 
-    console.log("Message saved succesfully:", doc);
+    // tried to simplify it the most, but  {..doc} did not work
+    await secondaryDatabase.create({ _id: doc._id, destination: doc.destination, body: doc.body, status: doc.status })
```

Comment:
The documents that come from mongoose bring a lot more info than it seems. The actual data is hidden under the hood so when you destructure a document you don't find what you expect